packages/react/src/views/LoginForm/LoginForm.js

```diff
@@ -52,7 +52,7 @@ export default function LoginForm() {
   };
 
   const handleEdituserOrEmail = (e) => {
-    setUserOrEmail(e.target.value);
+    setUserOrEmail(e.target.value.trim());
```

Comment:
@SinghaAnirban005  When we set the credential object and use the .trim() function there, why do we need it here as well?

Reply:
You’re absolutely right since I’ve already used .trim() when setting the credentials object, additional trimming is unnecessary. Thanks for the catch !!